yarn-project/end-to-end/src/cross_chain/test_harness.ts

```diff
@@ -161,12 +161,13 @@ export class CrossChainTestHarness {
   async checkEntryIsNotInOutbox(withdrawAmount: bigint): Promise<Fr> {
     this.logger('Ensure that the entry is not in outbox yet');
     const contractInfo = await this.aztecNode.getContractInfo(this.l2Contract.address);
-    // 0x00f714ce, selector for "withdraw(uint256,address)"
+    // 0xb460af94, selector for "withdraw(uint256,address,address)"
     const content = sha256ToField(
       Buffer.concat([
-        Buffer.from([0x00, 0xf7, 0x14, 0xce]),
+        Buffer.from([0xb4, 0x60, 0xaf, 0x94]),
         toBufferBE(withdrawAmount, 32),
         this.ethAccount.toBuffer32(),
+        EthAddress.ZERO.toBuffer32(),
```

Comment:
Why is the designated caller set to always be zero? Just because the end-to-end is not checking? I think it would be better if we want to use this more generally that we are also including it as an arg, could even default to zero, but seems weird to hardcode it.

Reply:
agree